static/defaults.config.php

```diff
@@ -650,6 +650,16 @@
 		// Setting 0 would allow maximum worker queues at all times, which is not recommended.
 		'worker_load_exponent' => 3,
 
+		// worker_max_duration (Array)
+		// Maximum runtime per priority. Worker processes that exceed this runtime will be terminated.
+		'worker_max_duration' => [
+			PRIORITY_CRITICAL   => 720,
+			PRIORITY_HIGH       => 10,
+			PRIORITY_MEDIUM     => 60,
+			PRIORITY_LOW        => 180,
+			PRIORITY_NEGLIGIBLE => 720
+		],
+
```

Comment:
Maybe an idea for later PR: Move `PRIORITY_*` constants to `Worker` class?

Reply:
I second that, like all constants should be moved out of of `boot.php`. I'd prefer if it was done before these naked constants are used in a node's configuration file beyond the defaults.

Reply:
It is never a problem to deprecate old constants, we did this before.

Reply:
When they only are used in code, it's fine, but when they're supposed to be used in user-edited config files, then we don't have complete control and we should tread even lighter.

Reply:
So to have both worlds united:
- constants used in configuration file, only deprecate them and name the new place
- constants used in other code, move them (including addon's code) to new place.

Can we agree on this?

Reply:
Apparently that's what @annando went with, keeping the global constants for the configuration file and creating new class constants?